src/i_video.c

```diff
@@ -1521,12 +1514,6 @@ static void I_InitGraphicsMode(void)
                 SDL_GetError());
     }
 
```

Comment:
This is probably still interesting.

Reply:
I restore it in https://github.com/fabiangreffrath/woof/pull/1492/commits/1f72366e616dfd6ad54b118280df5d55e002a009